packages/java/tests/spring/react-i18n/frontend/_views/BasicI18NView.tsx

```diff
@@ -0,0 +1,25 @@
+import { i18n, translate } from '@vaadin/hilla-react-i18n';
+import { TextField } from '@vaadin/react-components';
+
+export default function BasicI18NView() {
+  return (
+    <>
+      <section className="flex p-m gap-m items-end">
+        <TextField id='name' label={translate('basic.form.name.label')}/>
+        <TextField id='address' label={translate('basic.form.address.label')}/>
+        <TextField
+          id='language'
+          label='Language'
+          onValueChanged={async (e) => {
+            await i18n.setLanguage(e.detail.value);
+          }}
+        />
+        <span id="out">
+          <>Language: {i18n.language.value}</>
+          <p></p>
+          <>Resolved Language: {i18n.resolvedLanguage.value}</>
```

Comment:
```suggestion
          Language: {i18n.language.value}
          <br/>
          Resolved Language: {i18n.resolvedLanguage.value}
```
Probably shouldn't put paragraphs or line breaks into spans in general. Could use two different paragraphs if you want to clean it up.